www/form/inner.js

```diff
@@ -4260,12 +4258,49 @@ define([
                     }
                 }
             }
+            let shiftButtons;
+            let drag;
+            if (editable) {
+                drag = $('#toggle-drag-off').attr('class').indexOf('toggle-active') !== -1 ? false : true;
+            if (!drag) {
+                var upButton = h('button.cp-form-arrow', h('i.fa.fa-arrow-up',  {
+                    'title': Messages.moveItemUp,
+                    'aria-hidden': true
+                }));
+                var downButton = h('button.cp-form-arrow', h('i.fa.fa-arrow-down', {
+                    'title': Messages.moveItemDown,
+                    'aria-hidden': true
+                }));
+                var shiftBlock = function(direction) {
+                    var blockIndex = content.order.indexOf(uid);
+                    if (direction === 'up' && blockIndex > 0) {
+                        content.order.splice(blockIndex-1, 0, content.order.splice(blockIndex, 1)[0]);
+                    } else if (direction === 'down' && blockIndex < content.order.length-1) {
+                        content.order.splice(blockIndex+1, 0, content.order.splice(blockIndex, 1)[0]);
+                    }
+                    updateForm(framework, content, true);
```

Comment:
Since the "up" button is still visible on the first element and "down" button visible on the last one, it means we can click without triggering a change. In this case, it would be nice to avoid calling "updateForm" which, as mentioned earlier, can take some time on mobile with a big form.